src/vcpkg/binarycaching.cpp

```diff
@@ -951,14 +951,22 @@ namespace
         static constexpr StringLiteral m_accept_header = "Accept: application/json;api-version=6.0-preview.1";
     };
 
+    enum class CacheResult
```

Comment:
Can this use `RestoreResult` instead of making up a new value?

Reply:
Absolutely, I just didn't find that enum.